docs/usage.adoc

```diff
@@ -46,6 +46,33 @@ If you only want to use smart testing on `surefire` or `failsafe` plugin definit
 
 To do it you just need to set `smart.testing.apply.to` with either `surefire` or `failsafe` to just enable in one of them.
 
+=== Specifying Concrete Set of Test Classes
+
+During development, you may want to run a single test class or a set of concrete test classes repeatedly. 
+You can do this by setting the maven `test` property to specific test class(es).
+
+==== To Run a Single Test
+
+`$ mvn -Dtest=SampleTest test`
+
+NOTE: This configuration disables smart testing and executes only the specified sample test.
+
+==== To Run Multiple Test Classes
+
+You can choose to execute multiple test classes using patterns as 
+http://maven.apache.org/surefire/maven-surefire-plugin/examples/single-test.html[supported by Maven].
+
+`$ mvn -Dtest=RemoteReusable*Test test`
+
+This property can also be used in combination with smart testing's <<configuration#_modes, `selecting`>> and 
+<<configuration#_modes,`ordering`>> mode to choose specific test classes.
+
+When used with `ordering` mode, the specified tests are executed in the order defined by the smart testing 
+strategy used. If none is applicable to the strategy, they are executed one after the other in no specific order.
+
+When used with `selecting` mode, the specified tests that fit the defined strategy are executed.
+However, if none fulfil the strategy criteria, no tests would be executed resulting in build failure.
+
```

Comment:
There is missing the case when a user specifies multiple tests in the specific order:
-Dtest=TestOne,TestTwo